assessment/ASR_Validation/wald2017B/tests

```diff
@@ -0,0 +1,20 @@
+[Tests]
+  issues = '#89'
+  requirement = 'Blackbear shall be capable of appropriately modeling the expansion induced
```

Comment:
Input files for alkali-silica reaction validation tests shall have valid syntax.